### PR TITLE
updating the readme to add clarity to cert docs and developer testing

### PR DESCRIPTION
**Pull request by acornett21**
- Updating readme to point to official certification docs for clarity
- Updating the testing section so parters know these are not tests they run, but tests for developers.
- [JIRA](https://issues.redhat.com/browse/EET-3216)

### CODE REVIEW

**Comment by acornett21** [comment]
@taylorjordanNC Please take a look at let us know if you have more suggestions. Thanks.

**Comment by taylorjordanNC** [comment]
lgtm!

**Review by taylorjordanNC — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@
 This project is in active and rapid development! Many facets of this project are
 subject to change, and some features are not fully implemented.
```
Comment (skattoju) [inline README.md:15]: May be add a section header ex: `## Certification Documentation` or something like that ?

**Review by skattoju — CHANGES_REQUESTED** [review]
A section header might help it stand out ex: ## Certification Workflow Docs or something like that ?

File: README.md (hunk 1 above)
Comment (acornett21) [reply README.md:15]: I added a header with the same name that is used on the website.

**Comment by acornett21** [comment]
> This looks like it does more than just update for the official docs? Do we want to split this into two PRs?
@bcrochet Both of these asks were in the JIRA (which I added above). I've updated the title of this PR for clarity, since it only pulled the first half of the commit message.

**Review by bcrochet — APPROVED** [review]
/lgtm

**Pull request merged**